skada/_dasvm.py

```diff
@@ -0,0 +1,259 @@
+"""
+    dasvm estimator
+==========================================
+
+The dsvm method comes from [1].
+
+.. [1]  Domain Adaptation Problems: A DASVM Classification
+        Technique and a Circular Validation Strategy
+        Lorenzo Bruzzone, Fellow, IEEE, and Mattia Marconcini, Member, IEEE
+
+"""
+# Author: Ruben Bueno <ruben.bueno@polytechnique.edu>
+# dasvm implementation
+
+import numpy as np
+import math
+
+from skada.base import BaseEstimator
+from skada.utils import check_X_y_domain, source_target_split
+from sklearn.base import clone
+
+from sklearn.svm import SVC
+
+
+class DASVMEstimator(BaseEstimator):
+    """ DASVM Estimator:
+
+        Parameters
+        ----------
+        base_estimator : BaseEstimator
+            The estimator that will be used in the algorithm,
+            It is a SVC by default, but can use any classifier
+            equiped with a `decision_function` method
+        k: int>0
+            The number of points per classes that will be discarded/added
+            at each steps of the algorithm
+        Stop : int
+            The maximal number of iteration of the algorithm when using `fit`
+        """
+
+    __metadata_request__fit = {'sample_domain': True}
+    __metadata_request__transform = {'sample_domain': True, 'allow_source': True}
+
+    def __init__(
+            self,
+            base_estimator=None,
+            k=3,
+            Stop=1_000,
```

Comment:
check if there is a similar parameters in the toolbox but maybe `max_iter`. Try to not use capital letter